lib/FusionInventory/Agent/Task/Inventory/Generic/Dmidecode/Memory.pm

```diff
@@ -110,6 +110,22 @@ sub _getMemories {
                 $memory->{CAPACITY} = getCanonicalSize($1, 1024);
             }
 
+            if ($info->{'Part Number'}
+                    &&
+                $info->{'Part Number'} !~ /
+                    Not\s*Specified |
+                    NOT\s*AVAILABLE |
+                    Unknown         |
+                    Ser\s*Num       |
+                    None            |
+                    O\.E\.M\.       |
+                    DIMM            |
+                    Part\s*Number
+                /xi
+            ) {
+                $memory->{MODEL} = $info->{'Part Number'};
```

Comment:
As a response to my comment on the unittest file (see below), you may better use this code:
```suggestion
                $memory->{MODEL} = getSanitizedString(hex2char($info->{'Part Number'}));
```

Reply:
Done.
Although `0x4B0000000000000000000000000000000000` is converted to just "K". Should we keep such entries or ignore/drop?

Reply:
Don't worry about that, we can do nothing if Part Number is wrongly reported by the memory module or dmidecode. Maybe that was fixed in dmidecode.

Reply:
There is already a generic value blacklist in getDmidecodeInfos() function:
```
        next if
            $2 eq 'N/A'                        ||
            $2 eq 'Not Specified'              ||
            $2 eq 'Not Present'                ||
            $2 eq 'Unknown'                    ||
            $2 eq '<BAD INDEX>'                ||
            $2 eq '<OUT OF SPEC>'              ||
            $2 eq '<OUT OF SPEC><OUT OF SPEC>' ;
```
Many of the values used in this new specific blacklist seems to be duplicates, excepted for case differences.

I'd rather make the generic blacklist more exhaustive (it currently lacks 'O.E.M' and 'none' values), make it case unsensitive, and trim this specific blacklist to values only relevant in memory context.